assets/src/modules/Lizmap.js

```diff
@@ -76,10 +76,20 @@ export default class Lizmap {
                 // Add CRS:84 projection, same as EPSG:4326 but with ENU axis orientation
                 proj4.defs("CRS:84","+proj=longlat +datum=WGS84 +no_defs +type=crs");
                 register(proj4);
+                let wmsLayer = wmsCapabilities.Capability.Layer;
+                while (wmsLayer.BoundingBox === undefined) {
```

Comment:
I think I would add your PR description within the code as JS comment here. It might be easier to understand later (or better comment)

Reply:
Thanks @Gustry